tests/integration/main.go

```diff
@@ -131,9 +146,8 @@ func (tr *TestRun) runStep(step Step, verbose bool) {
 	}
 }
 
```

Comment:
line 133, 
log.Fatalf(fmt.Sprintf(`unknown action in testRun %s: %#v`, tr.name, action)) -> log.Fatalf(`unknown action in testRun %s: %#v`, tr.name, action) ?

Reply:
Changed :)